README.md

```diff
@@ -1,76 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Apple Page Cloned
 
-> One paragraph statement about the project.
+> Made as the fourth HTML/CSS project in the Microverse curriculum
```

Comment:
- [ ] Instead of saying its a project from microverse, you could write more about the project, and the technologies you used building it.